src/frontend/common/api.ts

```diff
@@ -727,6 +727,7 @@ export enum TickerPart {
 export interface clock {
   part: TickerPart;
   periodMs: number;
+  timeZone?: string | null;
```

Comment:
Now there is a strange type in the typescript.
But I guess we will fix it once we (cc @kunyavskiy) figure out a way to canonize them to all be either null or undefined.